test/integration/event-convert-data-test.js

```diff
@@ -0,0 +1,17 @@
+require('require-self-ref')
```

Comment:
I wish there was a way to have this applied to all of our test files. I wonder if `ava` has something built in that would allow that.